### ISSUE TITLE
Rosdep Error

### ISSUE THREAD

**Issue by mitsudome-r** [KIT-MRT/mrt_cmake_modules#2]
When I have this package under catkin_ws and run rosdep install, I get following error. 
```
mrt_cmake_modules: Cannot locate rosdep definition for [coverage]
```

Is this key for installing lcov and gcovr as defined in base.yaml file? 
https://github.com/KIT-MRT/mrt_cmake_modules/blob/032ea17c2c2aeb897a702c808f84f3a94ef283e7/yaml/base.yaml#L21

If that's the case, then wouldn't it be better to use lconv and gcovr as a key for dependency?

**Comment by mitsudome-r** [comment]
Are their any comments on this error? 
This error must be fixed in order to use this package in our project.

**Comment by poggenhans** [comment]
Fixed in e5e415f.
